readme.md

```diff
@@ -15,11 +15,17 @@ $ npm install --save dot-prop
 ```js
 var dotProp = require('dot-prop');
 
+// getter
 dotProp({foo: {bar: 'unicorn'}}, 'foo.bar');
 //=> 'unicorn'
 
-dotProp({foo: {bar: 'a'}}, 'foo.notDefined.deep');
```

Comment:
This should throw an error because `foo.bar` is `undefined`. If `dotProp({foo: {bar: {baz: null}}}, 'foo.baz.bar') === undefined` it implies `foo.baz` is not `undefined`